src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { FilterOption } from '../../types/FilterOption';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  activeTodos: Todo[];
+  filterOption: FilterOption;
+  setFilterOption: (option: FilterOption) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  activeTodos,
+  filterOption,
+  setFilterOption,
+}) => (
+  <footer className="todoapp__footer">
+    <span className="todo-count">
+      {`${activeTodos.length} items left`}
+    </span>
+
+    {/* Active filter should have a 'selected' class */}
+    <nav className="filter">
+      <a
+        href="#/"
+        className={classNames('filter__link',
+          { selected: filterOption === FilterOption.ALL })}
+        onClick={() => setFilterOption(FilterOption.ALL)}
+      >
+        All
+      </a>
```

Comment:
Would be great to create an array from options and iterate over it to not repeat the same logic several times.